Examples/River_area_test/Joiningmodels.json

```
@@ -0,0 +1 @@
+{"first_model_part_name": "Colors.SLIPBC", "second_model_part_name": "Colors.Walls2", "result_model_part_name": "Colors.SLIPBC", "boolean_operation": "Union", "entity_type": "Conditions"}
```

```suggestion
{
    "first_model_part_name": "Colors.SLIPBC",
    "second_model_part_name": "Colors.Walls2", 
    "result_model_part_name": "Colors.SLIPBC",
    "boolean_operation": "Union",
    "entity_type": "Conditions"
}
```
For the sake of clarity